scripts/chown_directories_for_postinstall.py

```diff
@@ -6,5 +6,10 @@
 if os.path.exists(paths.service_account_file_path()):
     service_account_uid = paths.irods_uid()
     service_account_gid = paths.irods_gid()
-    for dirpath, _, _ in os.walk(paths.irods_directory()):
+    is_top = True
+    for dirpath, dirnames, _ in os.walk(paths.irods_directory()):
+        if is_top:
```

Comment:
Just for my own understanding, this check is here to ensure that we are only skipping the "official" msiExecCmd_bin and not one that may be deeper down and was added by the administrators for whatever reason?

Reply:
Correct

Reply:
Cool, thanks!